addons/website_slides/static/src/js/slides_course_slides_list.js

```diff
@@ -52,12 +54,18 @@ publicWidget.registry.websiteSlidesCourseSlidesList = publicWidget.Widget.extend
      * @private
      */
     _checkForEmptySections: function (){
-        this.$('.o_wslides_js_slides_list_container ul').each(function (){
-            var $emptyCategory = $(this).find('.o_wslides_js_slides_list_empty');
-            if ($(this).find('li.o_wslides_slides_list_slide[data-slide-id]').length === 0) {
-                $emptyCategory.removeClass('d-none').addClass('d-flex');
-            } else {
-                $emptyCategory.addClass('d-none').removeClass('d-flex');
+        this.$('.o_wslides_slide_list_category').each(function (){
+            var $categoryHeader = $(this).find('.o_wslides_slide_list_category_header');
+            var categorySlideCount = $(this).find('.o_wslides_slides_list_slide:not(.o_not_editable)').length;
+            var $emptyFlagContainer = $($categoryHeader.find('.o_wslides_slides_list_drag')[0]);
```

Comment:
Why the `[0]` ?
Do you only want the first element?

If so you can use `.first()` jQuery method.